client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -304,15 +304,15 @@ class DomainRegistrationSuggestion extends React.Component {
 			progressBarProps = {
 				color: NOTICE_GREEN,
 				title,
-				value: matchScore * 100 || 90,
+				value: Math.min( matchScore * 100, 90 ),
```

Comment:
The end result looks a bit weird when I actually get an exact match (including the TLD I entered!), because it doesn't show a 100% match.

Might it make sense to adjust this to something more like the following?
```suggestion
				value: matchScore >= 1 ? 100 : Math.min( matchScore * 100, 90 ),
```

Reply:
Yeah, I was preserving the previous behaviour - IIRC there was a concern that if we show 100%, then it could be interpreted as "100% satisfaction guaranteed!". But we can try it 🤷‍♂️

Reply:
Maybe just limit it the full progress bar to exact matches for the SLD and TLD? It looks like we show a match reason for that case, so that might be the only edge case where it's worth it.

If we go that path, maybe merge this and iterate in a separate patch? (Though it may not be worth the effort of a separate PR and deploy! :P )

Reply:
Makes sense - I've adjusted the logic to show 100% if we have a SLD _and_ TLD exact match. Otherwise, fall back to 90/82.